Tools/autotest/arducopter.py

```diff
@@ -153,7 +153,11 @@ def land_and_disarm(self, timeout=60):
         """Land the quad."""
         self.progress("STARTING LANDING")
         self.change_mode("LAND")
-        self.wait_altitude(-5, 1, relative=True, timeout=timeout)
+        self.wait_land_and_disarm(timeout=60)
```

Comment:
Need to make this less like a verb.
```suggestion
        self.wait_landed_and_disarmed(timeout=60)
```

Reply:
done